### DISCUSSION TITLE
Cannot access time_locked_balance after force close

### DISCUSSION THREAD

**Discussion by cliffpeskin** [lightningnetwork/lnd#7288] [Troubleshooting]
**Background**
My node force closed a channel (https://mempool.space/lightning/channel/838007080834629633) and now after a couple months there is still a "time_locked_balance' of 1,596,149 sats that I believe was my node's balance at the time of closure.

**Question:**
How do I collect these sats?

**Environment**
LND v0.15.5-beta
Ubuntu Server 22.04.1 LTS
Bitcoin Core RPC client version v22.0

**Expected Behevior**
Time locked balance is swept into my wallet shortly after force closing a channel.

**Actual behavior**
lnd shows a 'time_locked_balance' of 1596149 for the channel 6 weeks after force close.

Channel output from `lncli closedchannels`:

```
"channel_point": "77b56b2dfd23905c942c1cee8259c2a64d5b40b4897f36a0dc4837ea0af0c24f:1",
            "chan_id": "838007080834629633",
            "chain_hash": "000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f",
            "closing_tx_hash": "306f5b730beeb852da80bcf2c88090f1ae83726d891e63eef5c07e69d32a51a6",
            "remote_pubkey": "035b1ff29e8db1ba8f2a4f4f95db239b54069cb949b8cde329418e2a83da4f1b30",
            "capacity": "3000000",
            "close_height": 763414,
            "settled_balance": "1278613",
            "time_locked_balance": "1596149",
            "close_type": "LOCAL_FORCE_CLOSE",
            "open_initiator": "INITIATOR_LOCAL",
            "close_initiator": "INITIATOR_LOCAL",
            "resolutions": [
                {
                    "resolution_type": "OUTGOING_HTLC",
                    "outcome": "TIMEOUT",
                    "outpoint": {
                        "txid_bytes": "6ca6a17198ecca4d8dfa005d2413d87ce00cdc349f215f02135a4f65568a8407",
                        "txid_str": "07848a56654f5a13025f219f34dc0ce07cd813245d00fa8d4dcaec9871a1a66c",
                        "output_index": 0
                    },
                    "amount_sat": "317536",
                    "sweep_txid": "f3719e7fd91c71320fd8337e1e6c0574516c2f92d2a4fb8e4226055fc80d1599"
                },
                {
                    "resolution_type": "OUTGOING_HTLC",
                    "outcome": "FIRST_STAGE",
                    "outpoint": {
                        "txid_bytes": "a6512ad3697ec0f5ee631e896d7283aef19080c8f2bc80da52b8ee0b735b6f30",
                        "txid_str": "306f5b730beeb852da80bcf2c88090f1ae83726d891e63eef5c07e69d32a51a6",
                        "output_index": 2
                    },
                    "amount_sat": "317536",
                    "sweep_txid": "07848a56654f5a13025f219f34dc0ce07cd813245d00fa8d4dcaec9871a1a66c"
                },
                {
                    "resolution_type": "COMMIT",
                    "outcome": "CLAIMED",
                    "outpoint": {
                        "txid_bytes": "a6512ad3697ec0f5ee631e896d7283aef19080c8f2bc80da52b8ee0b735b6f30",
                        "txid_str": "306f5b730beeb852da80bcf2c88090f1ae83726d891e63eef5c07e69d32a51a6",
                        "output_index": 3
                    },
                    "amount_sat": "1278613",
                    "sweep_txid": "f3719e7fd91c71320fd8337e1e6c0574516c2f92d2a4fb8e4226055fc80d1599"
                }
            ],
            "alias_scids": [
            ],
            "zero_conf_confirmed_scid": "0"
        },
```

**Comment by guggero** [comment]
It looks like the funds landed in your wallet a while ago: https://blockstream.info/tx/f3719e7fd91c71320fd8337e1e6c0574516c2f92d2a4fb8e4226055fc80d1599

A channel is only moved to `closedchannels` once it's fully processed. So everything looks normal. The `time_locked_balance` field only indicates _how much_ of the balance was affected by a time lock, it doesn't mean it still is time locked. The status of the individual components of that balance (the `1278613` sats of the `COMMIT` resolution and the `317536` sats of the `OUTGOING_HTLC`) are `CLAIMED` and `TIMEOUT` (which means the timeout script path was taken and since there's a sweep TX, that completed successfully).
So from what I can see all funds are properly accounted for and were available to your wallet (it looks like they were already used for opening another channel).

**Reply by cliffpeskin** [reply]
Thanks very much @guggero, that's super helpful and explains everything.
